src/postgres-util/src/desc.proto

```diff
@@ -19,9 +19,13 @@ message ProtoPostgresTableDesc {
 }
 
 message ProtoPostgresColumnDesc {
+    reserved 4, 5;
+    reserved "nullable", "primary_key";
     string name = 1;
     uint32 type_oid = 2;
     int32 type_mod = 3;
-    bool nullable = 4;
-    bool primary_key = 5;
+    // bool nullable = 4;
```

Comment:
I think it'd be a good idea to leave those in even if we don't currently use them because it will avoid us from having to do another migration in the future and we want to use them anyway. Especially the nullability information should be very easy to incorporate when we define the `RelationDesc` of the subsource in `pure.rs`

Reply:
I fixed this to keep and use the nullability information. The key information is confusing because it both isn't used and isn't useful on its own. If we ever want to support the notion of keys, we'll already have to write a migration to determine which columns are in the keys, so we can repopulate this then.

Reply:
Chatted with Petros about the primary key information and agreed to move the key information onto the table description.